src/main/java/com/cloudbees/jenkins/plugins/bitbucket/api/credentials/BitbucketClientCertificateAuthenticator.java

```diff
@@ -0,0 +1,35 @@
+package com.cloudbees.jenkins.plugins.bitbucket.api.credentials;
+
+import com.cloudbees.jenkins.plugins.bitbucket.api.BitbucketAuthenticator;
+import com.cloudbees.plugins.credentials.common.StandardCertificateCredentials;
+import org.apache.http.impl.client.HttpClientBuilder;
+import org.apache.http.ssl.SSLContextBuilder;
+import org.apache.http.ssl.SSLContexts;
+
+import javax.net.ssl.SSLContext;
+import java.security.KeyManagementException;
+import java.security.KeyStoreException;
+import java.security.NoSuchAlgorithmException;
+import java.security.UnrecoverableKeyException;
+
+public class BitbucketClientCertificateAuthenticator extends BitbucketAuthenticator<StandardCertificateCredentials> {
+
+    public BitbucketClientCertificateAuthenticator(StandardCertificateCredentials credentials) {
+        super(credentials);
+    }
+
+    @Override
+    public void configureBuilder(HttpClientBuilder builder) {
+        try {
+            builder.setSSLContext(buildSSLContext());
+        } catch (NoSuchAlgorithmException | UnrecoverableKeyException | KeyStoreException | KeyManagementException ignored) {
+
```

Comment:
I think we'll need to either pass a  `TaskListener` to these methods or log the exceptions. Certainly unacceptable to ignore and continue.

May even need to wrap as an `IOException` and throw